dev/devicelab/lib/tasks/sample_catalog_generator.dart

```diff
@@ -19,7 +21,8 @@ Future<TaskResult> samplePageCatalogGenerator(String authorizationToken) async {
   await inDirectory(catalogDirectory, () async {
     await flutter('packages', options: <String>['get']);
 
-    if (deviceOperatingSystem == DeviceOperatingSystem.ios)
+    final bool isIOSDevice = deviceOperatingSystem == DeviceOperatingSystem.ios;
```

Comment:
nit: for consistency with other code, let's treat "ios" as a plain word and camel-case it like this: `isIosDevice`.